apps/coscrad-frontend/src/components/live/live.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { screen } from '@testing-library/react';
+import { renderWithProviders } from '../../utils/test-utils';
+import { getDummyConfigurableContent } from '../../utils/test-utils/get-dummy-configurable-content';
+
+import { Live } from './live';
+
+describe('Live', () => {
+    it('should render successfully', () => {
+        const { baseElement } = renderWithProviders(<Live />);
+        expect(baseElement).toBeTruthy();
+    });
+
+    it('should load configurable content', () => {
+        const dummyConfigurableContent = getDummyConfigurableContent();
+        const listenLive = 'Listen Live';
```

Comment:
This will have to be updated after you make this an object. But good test. This is the right pattern.

Reply:
The test has been updated.